check-digit/src/js/Form.jsx

```diff
@@ -42,7 +42,8 @@ class Form extends Component {
     const label = {
```

Comment:
The label object should be moved out of render, either as a free-standing object in the module, or into the constructor as a property. Otherwise this static object is being redefined every render